frontend/src/styles/mixins/_classic-theme.scss

```
@@ -1,58 +1,89 @@
 // Classic: the light palette. Every colour the app paints comes from this list,
 // so a theme is a token swap and never a per-component patch list.
+// The palette follows Gmail's light look: a cool grey page, white list cards,
+// blue as the single accent, and shaded read rows against white unread ones.
 @mixin theme {
   color-scheme: light;
   --font-ui: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
-  --font-display: "Fraunces", Georgia, "Times New Roman", serif;
-  --font-brand: var(--font-display);
+  --font-display: "Google Sans", "Product Sans", Roboto, system-ui, -apple-system, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)